engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobContext.java

```diff
@@ -68,6 +73,7 @@
     private final String _analysisJobXml;
     private final String _analysisJobXmlPath;
     private final Map<String, String> _customProperties;
+    private final JavaSparkContext _sparkContext;
```

Comment:
You cannot keep the spark context in this class, since it needs to be serializable.